services/demo/src/view/forms/NestedValuesForm.tsx

```diff
@@ -45,41 +62,69 @@ export const NestedValuesForm: React.FC<NestedValuesFormProps> = ({
                     />
                 </Grid>
                 {showInputs && (
-                    <Grid container spacing={2}>
-                        <Grid item xs={12}>
-                            <TextField
-                                name="username"
-                                label="Username"
-                                validatorData={{
-                                    message:
-                                        'For some reason this field always has an error',
-                                    context: ValidatorContext.Danger,
-                                }}
-                                pristine={false}
-                                required
-                            />
-                        </Grid>
-                        <Grid item xs={12}>
-                            <NestedForm name="personal">
-                                <Grid container spacing={2}>
-                                    <Grid item xs={6}>
-                                        <TextField
-                                            name="firstName"
-                                            label="First name"
-                                            required
-                                        />
-                                    </Grid>
-                                    <Grid item xs={6}>
-                                        <TextField
-                                            name="lastName"
-                                            label="Last name"
-                                            required
-                                        />
+                    <React.Fragment>
+                        <Grid container spacing={2}>
+                            <Grid item xs={12}>
+                                <TextField
+                                    name="username"
+                                    label="Username"
+                                    validatorData={{
+                                        message:
+                                            'For some reason this field always has an error',
+                                        context: ValidatorContext.Danger,
+                                    }}
+                                    pristine={false}
+                                    required
+                                />
+                            </Grid>
+                            <Grid item xs={12}>
+                                <NestedForm name="personal">
```

Comment:
I added a demo here to demostrate the issue I got in Prime+, we have a `<NestedForm>` and we have a mirror to reflect that `<NestedForm>`, the mirror works if I manually change the nested form fields, but it doesn't work if I change that prograatically by clicking this button.

After I added this demo, it works in this repo, then I realised that it works because we have a state `mirroredFormData` in `App.tsx` to do a render, that re-render will trigger a render for `Mirror`, so I commented out that logic in `App.tsx` to just replicate this issue. 

The proposed fix is to add back the `_handleUpdate` in `NestedForm` (the one I commented out in this PR), I noticed that part of code is in master branch actually, but it was deleted in this branch, if you restore that part it will work.